addons/hr/views/res_users.xml

```diff
@@ -43,90 +22,41 @@
         <record id="res_users_view_form_profile" model="ir.ui.view">
             <field name="name">res.users.preferences.form.inherit</field>
             <field name="model">res.users</field>
-            <field name="inherit_id" ref="res_users_view_form_simple_modif"/>
+            <field name="inherit_id" ref="base.view_users_form_simple_modif"/>
```

Comment:
I would take the opportunity of this file to clarify in a comment what views are used when / where / based on which ones. (just a few lines to clarify)

Reply:
(hr vs preferences)